js/index.js

```diff
@@ -0,0 +1,60 @@
+const bookForm = document.querySelector('#book-form');
+const booksList = document.getElementById('book-list');
+const bookTitle = document.querySelector('#book-title');
+const bookAuthor = document.querySelector('#book-author');
+
+const booksData = [];
+
+function AddBook(book) {
+  const bookElement = `
+    <li class="book">
+      <h5>${book.title}</h5>
+      <h6>${book.author}</h6>
+      <button data-id="${book.id}" class="book-remove">Remove</button>
+    </li>`;
+  booksList.innerHTML += bookElement;
+}
+
+function storageWriter() {
+  localStorage.setItem('books', JSON.stringify(booksData));
+}
+
+bookForm.addEventListener('submit', () => {
+  const book = {
+    id: Math.round(Math.random() * 10000000),
+    title: bookTitle.value,
+    author: bookAuthor.value,
+  };
+  AddBook(book);
+  booksData.push(book);
+  storageWriter();
+  bookTitle.value = '';
+  bookAuthor.value = '';
+});
+
+window.addEventListener('load', () => {
+  const books = JSON.parse(localStorage.getItem('books'));
+  for (let i = 0; i < books.length; i += 1) {
+    booksData.push(books[i]);
+    AddBook(books[i]);
+  }
+});
```

Comment:
- [ ] [OPTIONAL] when I load the page this error appears on the console, and this because at the beginning when you try to get the books at line 36, there's no books at this time, so it gives error when you try to get the length, to fix it you can check before the loop and if the books = undefined you can set the books with empty array. 

![image](https://user-images.githubusercontent.com/35535130/163992827-07bfe8df-4a5f-49a5-a9ee-e6d838e041ad.png)